src/utils.js

```diff
@@ -452,29 +452,19 @@ exports.insertElement = function(elm, doc, target) {
   } catch (e) {}
 };
 
-exports.insertPixel = function (url) {
+exports.triggerPixel = function (url) {
   const img = new Image();
-  img.id = _getUniqueIdentifierStr();
   img.src = url;
-  img.height = 0;
-  img.width = 0;
-  img.style.display = 'none';
-  img.onload = function() {
-    try {
-      this.parentNode.removeChild(this);
-    } catch (e) {
-    }
-  };
-  exports.insertElement(img);
 };
 
 /**
  * Inserts empty iframe with the specified `url` for cookie sync
  * @param  {string} url URL to be requested
  * @param  {string} encodeUri boolean if URL should be encoded before inserted. Defaults to true
  */
-exports.insertCookieSyncIframe = function(url, encodeUri) {
-  let iframeHtml = this.createTrackPixelIframeHtml(url, encodeUri);
+exports.insertUserSyncIframe = function(url) {
+  let iframeHtml = this.createTrackPixelIframeHtml(url, false);
+  iframeHtml = iframeHtml.replace('<iframe', '<iframe sandbox="allow-scripts"');
```

Comment:
String parsing is very fragile... would suggest one of the following instead:

- Change `createTrackPixelIframeHtml` so that it accepts a 'sandbox' option, and builds the right HTML from it
- Create the `div` object first, and then set the sandbox attribute programatically